Quantized interpolation benchmarks

z-a-f opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#29509 Quantized interpolation benchmarks**

Differential Revision: [D18415367](https://our.internmc.facebook.com/intern/diff/D18415367)

Commit:
Quantized interpolation benchmarks

mingzhe09088 commented on benchmarks/operator_benchmark/pt/qinterpolate_test.py:
```diff
@@ -0,0 +1,64 @@
+the `interpolate` anyway.
+'''
+
+# Configs for PT add operator
```
Modify the comment?

mingzhe09088 commented on benchmarks/operator_benchmark/pt/qinterpolate_test.py:
```diff
@@ -0,0 +1,64 @@
+    def init(self, M, N, K, dtype, mode, scale):
+        f_input = (torch.rand(1, M, N, K) - 0.5) * 256
+        scale = 0.1
+        zero_point = 64
```
Just want to check if zero_point is intentionally set to 64? As I saw it was set to 0 for other tests.

z-a-f replied:
Yes, my logic was in case there is a fast pass in the interpolation logic, where everything is 0, the behavior might be different. However, if the dtype is `quint8`, all the negative numbers might become 0, which will take a fast pass. To avoid that, I am setting the zero_point to some non-zero value.

I cannot set it to 128, as that would have the same (potential) problem with `qint8`, hence 64... I guess I should set it to 42 :)

Commit:
Update on "Quantized interpolation benchmarks"


Differential Revision: [D18415367](https://our.internmc.facebook.com/intern/diff/D18415367)

Commit:
Update on "Quantized interpolation benchmarks"


Differential Revision: [D18415367](https://our.internmc.facebook.com/intern/diff/D18415367)

Commit:
Added contiguous test on "Quantized interpolation benchmarks"


Differential Revision: [D18415367](https://our.internmc.facebook.com/intern/diff/D18415367)

raghuramank100 commented on benchmarks/operator_benchmark/pt/qinterpolate_test.py:
```diff
@@ -0,0 +1,68 @@
+    cross_product_configs={
+        'dtype': [torch.quint8, torch.qint8, torch.qint32],
+        'mode': ['nearest', 'bilinear'],
+        'scale': [0.25, 0.5, 1.0, 1.5, 2.0],
```
Suggest reducing this to reduce the number of tests
```
'scale': [0.5, 1.0, 2.0]
```

raghuramank100 reviewed (approved):
Looks good, please consider reducing number of test cases

Commit:
Update on "Quantized interpolation benchmarks"


Differential Revision: [D18415367](https://our.internmc.facebook.com/intern/diff/D18415367)
